schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-activites.js

```diff
@@ -38,6 +43,22 @@
             $location.path('/admin/activities/' + organization.id);
         }
 
+        function queryActivities() {
+            if (vm.organization) {
+                var dfd = $q.defer();
+
+                Activity.query({
+                    organizationId: vm.organization.id,
+                    fromDate: vm.activitiesQueryFromDate
+                }).then(function (result) {
+                    dfd.resolve(angular.extend({items: []}, result));
+                    vm.activities = result.items;
+                });
+            } else {
+                return {items: []};
```

Comment:
This is wrong.  You are returning what to whoom?
Instead of 
```javascript
    return {items:[]};
```
You need:
```javascript
vm.activities = [];
```

I guess you copied it from the routes...